support/extract-utils.js

```diff
@@ -22,8 +23,8 @@ const DOCUMENT_PUBLISHED_STATUS = 'http://mu.semte.ch/application/concepts/ef8e4
  * an extract is the treatment of one agendapoint and all it's related info
  */
 
-async function buildExtractForTreatment(treatment, meeting, meetingErrors) {
-  const data = await buildExtractDataForTreatment(treatment, meeting, true);
+async function buildExtractForTreatment(treatment, meeting, meetingErrors, participantCache) {
```

Comment:
I don't think this function needs this parameter and certainly not as a required one.

Reply:
It's only called one line 52
```
const extractBuilders = treatments.map( (treatment) => buildExtractForTreatment(treatment, meeting, meetingErrors, participantCache) );
```

And it passes it, I will make it optional.
It needs it because it's passing it down to `buildExtractDataForTreatment` who is passing it down to `fetchTreatmentParticipantsWithCache`.